circle.yml

```diff
@@ -0,0 +1,15 @@
+machine:
+  environment:
+    COVERALLS_REPO_TOKEN: LcPzw7iQRxVIRBcnQWXlZgegxcdxDabg9
```

Comment:
Is it secure to have this token here? Or should we add it as an environment variable in CircleCI?